cloudformation/cloudformation.yaml

```diff
@@ -8,7 +8,109 @@ Parameters:
             - PROD
             - CODE
         Default: CODE
+Mappings:
+  EnvironmentBucketMap:
+    CODE:
+      "logs": "fulfilment-s3-logs-code"
+      "salesforce": "fulfilment-salesforce-backup-code"
+      "zuora": "fulfilment-zuora-staging-code"
+      "export": "fulfilment-export-code"
+    PROD:
+      "logs": "fulfilment-s3-logs-prod"
+      "salesforce": "fulfilment-salesforce-backup-prod"
+      "zuora": "fulfilment-zuora-staging-prod"
```

Comment:
It would be good to document the uses for these different buckets in the README, as it's not entirely intuitive to someone unfamiliar with this project (feel free to do that in a separate PR)